components/FindResidents/Resident/ResidentPage/index.js

```diff
@@ -56,50 +56,63 @@ const ResidentPage = ({
           style={styles.profPic}
           source={pictureUrl}
         />
-        <View style={styles.nameContainer}>
-          <Title>{`${fname} ${lname}`}</Title>
+        <View style={{ margin: 7 }}>
+          <View style={styles.nameContainer}>
+            <Text style={[styles.name, { fontWeight: 'bold' }]}>{`${fname} ${lname}`}</Text>
+          </View>
           <Text style={styles.name}>{`"${nickname}"`}</Text>
-          <Button
+
+          {/* <Button
             style={styles.button}
             contentStyle={styles.buttonContent}
           >
             {I18n.t('findResident.residentPage.household.editProfile')}
-          </Button>
+          </Button> */}
         </View>
       </View>
       <View
         style={styles.horizontalLine}
       />
-      <View style={styles.navigationButtons}>
-        <Button onPress={() => showDemographics()}>{I18n.t('findResident.residentPage.household.demographics')}</Button>
-        <Button onPress={() => showForms(true)}>{I18n.t('findResident.residentPage.household.forms')}</Button>
-        <Button onPress={() => showHousehold(true)}>{I18n.t('findResident.residentPage.household.household')}</Button>
+      <View style={styles.navigationButtonsContainer}>
+        <Button style={styles.navigationButton} labelStyle={styles.navigationButtonText} onPress={() => showDemographics()}>{I18n.t('findResident.residentPage.household.demographics')}</Button>
+        <Button style={styles.navigationButton} labelStyle={styles.navigationButtonText} onPress={() => showForms(true)}>{I18n.t('findResident.residentPage.household.forms')}</Button>
+        {/* <Button style={styles.navigationButton}
+        labelStyle={styles.navigationButtonText}
+        onPress={() => showHousehold(true)}>
```

Comment:
We'll still need to have the householdId working offline as well, I assume you commented it out temporarily right?